src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd

```diff
@@ -0,0 +1,191 @@
+@echo off
+setlocal
+
+set Id=%1
+set IdMaster=Master
+set WcfTestDir=c:\WCFTest
+set RepoHome=c:\git
```

Comment:
Question: could we infer RepoHome from %_dp0?  Or is it the intent that this CMD will be moved out of the repo and executed somewhere else?